### DISCUSSION TITLE
Глюки у ГГ

### DISCUSSION THREAD

**Discussion by behowtour** [behowtour/ITS#139] [Ideas]
Некоторое время как в калейдоскопе

**Comment by behowtour** [comment]
Типа что-то на пути висит - натыкаешься, всё переходит в слоу-мо и есть две секунды чтобы принять решение - юзать или нет. И типа если юзаешь то как "под лсд" - всё такое как в калейдоскопе становится и там какие-то свои поинты и всё такое. И это типа мини испытание, по его итогу какой-то бонус получаешь, но и какой-то дебаф или штраф или откат в любом случае получаешь, поэтому и надо решать типа брать или не брать эту штуку.

**Comment by behowtour** [comment]
@Sparrowack  как тебе?

**Reply by Sparrowack** [reply]
очень интересная может быть вещь, как площадка для опробывания механик, например наведение вектора выброса веревки
